appmap/test/test_django.py

```diff
@@ -1,25 +1,18 @@
 # flake8: noqa: E402
-# pylint: disable=unused-import
-
-import pytest
-
-pytest.importorskip('django')
+# pylint: disable=unused-import, redefined-outer-name, missing-function-docstring
 
 import django.conf
 import django.db
+import django.http
 import django.test
+import django.urls
+import pytest
+
+
 import appmap.django  # noqa: F401
 
+from .web_framework import TestRequestCapture, TestRecording
 
-# Note that with django configured this way, django.test.Client
-# requests for '/test' will print warning messages that look like
-# `WARNING:django.request:Not Found: /test`. These don't appear to
```

Comment:
You moved this code, but not the comment. Have you made other changes that got rid of these warnings (when you run `pytest -svv appmap`? If not, can you put the comment (or some variation of it) back, please?

Reply:
Yes, I added the views and routes.